### PR TITLE
Add Dockerfile and documentation

### PR DESCRIPTION
**Pull request by sportebois**
As explained in the README, the goal is to provide the easiest setup possible for developers to get a  RabbitMQ cluster running, with the same version and plugins that their code will later use on CloudAMQP.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,46 @@
...
+- rabbitmq_delayed_message_exchange
+- rabbitmq_management_agent
+- rabbitmq_web_dispatch
+- webmachine
+- mochiweb
+
+
+## Use it
+
+Baked-it settings, using docker-compose:
```
Comment (mspensieri) [inline README.md]: Baked-in ?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,46 @@
...
+
+    docker run -it --rm --hostname local-rabbit -p 15672:15672 -p 5672:5672 smooch/rabbitmq
+
+
+If you don't have yet any code to test it, you can already access the management UI to check that your RabbitMQ service is running.
+
+
+## Open management plugin
+
+The management plugin is exposed using the default port: [http://guest:guest@localhost:15672/#/](http://localhost:15672/)
```
Comment (mspensieri) [inline README.md]: I think this markdown link is backwards. Friendly text goes in `[]` and full url goes in `()`

Comment (sportebois) [inline README.md]: Indeed... it's obvious with text and link. It's less obvious with link+link and I guess I was tired ;D

File: README.md (hunk 1 above)
Comment (sportebois) [inline README.md]: ... I did installed a spell-checker, but didn't helped for this one though !

**Pull request merged**